mmedit/engine/runner/__init__.py

```diff
@@ -1,9 +1,10 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+from .edit_loops import EditTestLoop, EditValLoop
 from .gen_loops import GenTestLoop, GenValLoop
-from .log_processor import GenLogProcessor
+from .log_processor import EditLogProcessor
 from .multi_loops import MultiTestLoop, MultiValLoop
 
 __all__ = [
-    'MultiValLoop', 'MultiTestLoop', 'GenTestLoop', 'GenValLoop',
-    'GenLogProcessor'
+    'EditTestLoop', 'EditValLoop', 'MultiValLoop', 'MultiTestLoop',
```

Comment:
`EditxxLoop` and `GenxxLoop` are not intuitive enough. 
Can we merge them? Or naming them in a better way.